### ISSUE TITLE
[Beta] Applied Accessibility - Tests fail to check for closing </nav> tag

### ISSUE THREAD

**Issue by Asjas** [freeCodeCamp/freeCodeCamp#16415]
#### Challenge Name

https://beta.freecodecamp.org/en/challenges/applied-accessibility/make-screen-reader-navigation-easier-with-the-nav-landmark

#### Issue Description

There are two issues I've found here.

Firstly, One of the tests says there shouldn't be any `<div>` tags used in this code. If a closing `</div>` tag is left in the code the test passes.

Secondly, If you use a opening `<nav>` tag and leave out the closing `</nav>` tag the test also passes.

#### Browser Information

* Browser Name, Version: Chrome, Version 63.0.3239.108 (Official Build) (64-bit)
* Operating System: Windows 10 Pro
* Mobile, Desktop, or Tablet: Desktop

#### Your Code

```js
<body>
  <header>
    <h1>Training with Camper Cat</h1>

    <nav>
      <ul>
        <li><a href="#stealth">Stealth &amp; Agility</a></li>
        <li><a href="#combat">Combat</a></li>
        <li><a href="#weapons">Weapons</a></li>
      </ul>
    </div>

  </header>
  <main>
    <section id="stealth">
      <h2>Stealth &amp; Agility Training</h2>
      <article><h3>Climb foliage quickly using a minimum spanning tree approach</h3></article>
      <article><h3>No training is NP-complete without parkour</h3></article>
    </section>
    <section id="combat">
      <h2>Combat Training</h2>
      <article><h3>Dispatch multiple enemies with multithreaded tactics</h3></article>
      <article><h3>Goodbye world: 5 proven ways to knock out an opponent</h3></article>
    </section>
    <section id="weapons">
      <h2>Weapons Training</h2>
      <article><h3>Swords: the best tool to literally divide and conquer</h3></article>
      <article><h3>Breadth-first or depth-first in multi-weapon training?</h3></article>
    </section>
  </main>
</body>
```
#### Screenshot

![accessibility](https://user-images.githubusercontent.com/3828967/34630795-43787aea-f276-11e7-8208-ec39dafe3255.jpg)

**Comment by matthras** [comment]
I've got one or two ideas for a fix - will be testing some changes locally for the next few days before I submit a pull request.

**Comment by anhtran1906** [comment]
I could reproduce the problems and successfully added tests for the "</div>" and "</nav>" tag. My solution is that I added two new lines of test to test if the user's answer includes "</nav>" and does not include "</div>". @raisedadead please let me know if you're OK with this solution and I can create a pull request within the next 8 hours

**Comment by raisedadead** [comment]
@matthras please reach out to [Contributors Chat room](https://gitter.im/FreeCodeCamp/Contributors), we will try and get your local setup up.

@anhtran1906 thanks for working on this. Please go ahead with a pull request, and we will guide with further feedback if needed.

Thanks and Happy contributing!
